examples/with-ava/.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [
+    "backpack-core/babel",
+    "stage-0",
```

Comment:
I think AVA comes with stage-0 etc. built-in. So I'm not sure this is necessary.